core/config.go

```diff
@@ -173,3 +183,10 @@ func WithListenAddress(addr string) ConfigOption {
 		d.listenAddr = addr
 	}
 }
+
+// WithListenPort specifies the address localhost:port the drand instance should bind to.
+func WithListenPort(port string) ConfigOption {
```

Comment:
It is much confusing of what port are you talking about, the regular drand port or the control port ? 
It should be clearer from the name of the method, like `WithControlPort` (same for the method that returns it).